subsys/net/lib/sockets/Kconfig

```diff
@@ -73,6 +73,31 @@ config NET_SOCKET_MAX_SEND_WAIT
 	  The maximum time a socket is waiting for a blocked connection before
 	  returning an ENOBUFS error.
 
+config NET_SOCKETS_SERVICE
+	bool "Enable sockets service support [EXPERIMENTAL]"
+	select EXPERIMENTAL
+	select EVENTFD
```

Comment:
I've had issues in the past with `EVENTFD` on some boards (I think it was `native_posix`), for that reason I've used `NET_SOCKETPAIR` in the CoAP server implementation.

Not sure if this is still an issue though.

Reply:
There was some issues with `native_sim`, I will investigate this more.

Reply:
Hmm, in eventfd Kconfig there is `depends on !NATIVE_APPLICATION` and the new `echo-service` process does not properly handle the incoming traffic with this board. I would like to avoid adding new dependency to socketpair just for this.